raiden/network/transport/matrix/utils.py

```diff
@@ -251,6 +251,22 @@ def _presence_listener(self, event: Dict[str, Any]):
         if self._user_presence_changed_callback:
             self._user_presence_changed_callback(user, new_state)
 
+    def get_status_info(self):
```

Comment:
maybe call this function `get_matrix_status_info` ? Or which status are you interested in?